src/ComposerRequireChecker/NodeVisitor/DefinedSymbolCollector.php

```diff
@@ -48,7 +48,7 @@ public function enterNode(Node $node)
 
     private function recordClassDefinition(Node $node)
     {
-        if ($node instanceof Node\Stmt\Class_) {
+        if ($node instanceof Node\Stmt\Class_ && !$node->isAnonymous()) {
```

Comment:
Is this API available in all versions of PHP-Parser? Just making sure we don't support older versions that don't have this.